pkg/client/eventsbuilder.go

```diff
@@ -32,14 +32,26 @@ func (builder *eventsBuilder) getStartPosition() *orderer.SeekPosition {
 
 type eventOption = func(builder *eventsBuilder) error
 
-// Checkpointer provides the current position for event processing.
-type Checkpointer interface {
+// Checkpoint provides the current position for event processing.
+type Checkpoint interface {
 	// BlockNumber in which the next event is expected.
 	BlockNumber() uint64
 	// TransactionID of the last successfully processed event within the current block.
 	TransactionID() string
 }
 
+// Checkpointer provides the current position for event processing.
+type Checkpointer interface {
+	// CheckpointBlock checkpoints the block number.
+	CheckpointBlock(uint64) error
+	// CheckpointTransaction checkpoints the transaction within a block.
+	CheckpointTransaction(uint64, string) error
+	// CheckpointChaincodeEvent checkpoints the event
```

Comment:
Need to be well-formed statement, with ending punctuation.
```suggestion
	// CheckpointChaincodeEvent checkpoints the chaincode event.
```